tests/components/demo/test_media_player.py

```diff
@@ -187,36 +194,42 @@ def test_prev_next_track(self):
 
     def test_play_media(self):
         """Test play_media ."""
-        assert setup_component(
-            self.hass, mp.DOMAIN, {"media_player": {"platform": "demo"}}
-        )
+        assert setup_component(self.hass, mp.DOMAIN,
+                               {"media_player": {
+                                   "platform": "demo"
+                               }})
         ent_id = "media_player.living_room"
         state = self.hass.states.get(ent_id)
-        assert 0 < (mp.SUPPORT_PLAY_MEDIA & state.attributes.get("supported_features"))
+        assert 0 < (mp.SUPPORT_PLAY_MEDIA
+                    & state.attributes.get("supported_features"))
         assert state.attributes.get("media_content_id") is not None
 
         with pytest.raises(vol.Invalid):
             common.play_media(self.hass, None, "some_id", ent_id)
         self.hass.block_till_done()
         state = self.hass.states.get(ent_id)
-        assert 0 < (mp.SUPPORT_PLAY_MEDIA & state.attributes.get("supported_features"))
+        assert 0 < (mp.SUPPORT_PLAY_MEDIA
+                    & state.attributes.get("supported_features"))
         assert not "some_id" == state.attributes.get("media_content_id")
 
         common.play_media(self.hass, "youtube", "some_id", ent_id)
         self.hass.block_till_done()
         state = self.hass.states.get(ent_id)
-        assert 0 < (mp.SUPPORT_PLAY_MEDIA & state.attributes.get("supported_features"))
+        assert 0 < (mp.SUPPORT_PLAY_MEDIA
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)